src/sourcemaps/tools/rollup.ts

```diff
@@ -0,0 +1,69 @@
+// @ts-ignore - clack is ESM and TS complains about that. It works though
+import clack, { select } from '@clack/prompts';
+import chalk from 'chalk';
+import {
+  abortIfCancelled,
+  addDotEnvSentryBuildPluginFile,
+  getPackageDotJson,
+  hasPackageInstalled,
+  installPackage,
+} from '../../utils/clack-utils';
+
+import {
+  SourceMapUploadToolConfigurationFunction,
+  SourceMapUploadToolConfigurationOptions,
+} from './types';
+
+const getCodeSnippet = (options: SourceMapUploadToolConfigurationOptions) =>
+  chalk.gray(`
+  ${chalk.greenBright(
+    'import { sentryRollupPlugin } from "@sentry/rollup-plugin";',
+  )}
+
+  export default {
+    output: {
+      ${chalk.greenBright(
+        'sourcemap: true, // Source map generation must be turned on',
+      )}
+    },
+    plugins: [
+      // Put the Sentry rollup plugin after all other plugins
+      ${chalk.greenBright(`sentryRollupPlugin({
+        authToken: process.env.SENTRY_AUTH_TOKEN,
+        org: "${options.orgSlug}",
+        project: "${options.projectSlug}",${
+        options.selfHosted ? `\n      url: "${options.url}",` : ''
+      }
+      }),`)}
+    ],
+  };
```

Comment:
We should probably remove the base level indentation of this snippet.

Reply:
Good catch, copy 🍝